recipes-bsp/opensbi/opensbi_0.3.bb

```diff
@@ -10,11 +10,10 @@ inherit autotools-brokensep
 
 SRC_URI = "https://github.com/riscv/opensbi/archive/v${PV}.tar.gz \
```

Comment:
Usually I would prefer a proper release or using git URI, github rebuilds the archives and it can end up in updated checksums silently.

Reply:
I can change it to a git clone if that's better. At the moment the "official" release is via the GitHub archives though

Reply:
I think its better to use SHA1 and  git fetcher instead